config/requirements.txt

```diff
@@ -1,45 +1,66 @@
 apache-libcloud==2.5.0
 asn1crypto==0.24.0
-azure-cli-core==2.0.35
-azure-graphrbac==0.40.0
-azure-keyvault==1.0.0a1
-azure-mgmt-authorization==0.51.1
+azure-cli-core==2.34.0
+azure-common==1.1.11
+azure-containerregistry==1.1.0
+azure-graphrbac==0.61.1
+azure-identity==1.7.0
+azure-keyvault==1.1.0
+azure-mgmt-apimanagement==3.0.0
+azure-mgmt-authorization==2.0.0
+azure-mgmt-automation==1.0.0
 azure-mgmt-batch==5.0.1
-azure-mgmt-cdn==3.0.0
-azure-mgmt-compute==4.4.0
-azure-mgmt-containerinstance==1.4.0
-azure-mgmt-containerregistry==2.0.0
-azure-mgmt-containerservice==4.4.0
-azure-mgmt-cosmosdb==0.5.2
-azure-mgmt-devtestlabs==3.0.0
-azure-mgmt-dns==2.1.0
-azure-mgmt-hdinsight==0.1.0
-azure-mgmt-keyvault==1.1.0
-azure-mgmt-loganalytics==0.2.0
-azure-mgmt-marketplaceordering==0.1.0
-azure-mgmt-monitor==0.5.2
-azure-mgmt-network==2.3.0
+azure-mgmt-cdn==11.0.0
+azure-mgmt-compute==26.1.0
+azure-mgmt-containerinstance==9.0.0
+azure-mgmt-containerregistry==9.1.0
+azure-mgmt-containerservice==20.0.0
+azure-mgmt-core==1.3.0
+azure-mgmt-cosmosdb==6.4.0
+azure-mgmt-datafactory==2.0.0
+azure-mgmt-datalake-store==1.0.0
+azure-mgmt-devtestlabs==9.0.0
+azure-mgmt-dns==8.0.0
+azure-mgmt-eventhub==10.1.0
+azure-mgmt-hdinsight==9.0.0
+azure-mgmt-iothub==2.2.0
+azure-mgmt-keyvault==10.0.0
+azure-mgmt-loganalytics==12.0.0
+azure-mgmt-managedservices==6.0.0
+azure-mgmt-managementgroups==1.0.0
+azure-mgmt-marketplaceordering==1.1.0
+azure-mgmt-monitor==3.0.0
+azure-mgmt-network==19.1.0
+azure-mgmt-notificationhubs==7.0.0
 azure-mgmt-nspkg==2.0.0
-azure-mgmt-rdbms==1.4.1
-azure-mgmt-redis==5.0.0
-azure-mgmt-resource==2.1.0
-azure-mgmt-servicebus==0.5.3
-azure-mgmt-sql==0.10.0
-azure-mgmt-storage==3.1.0
-azure-mgmt-trafficmanager==0.50.0
-azure-mgmt-web==0.41.0
-azure-storage==0.35.1
```

Comment:
this was dropped (think replaced by azure-storage-blob

Don't see `azure-storage` listed in https://learn.microsoft.com/en-us/python/api/overview/azure/storage?view=azure-python